ArcadiaParties-SPA/src/app/header/header.component.css

```diff
@@ -0,0 +1,7 @@
+.example-icon {
```

Comment:
You should use scss files to declare styles. It allows to use much more features than raw css.

Reply:
Changed CSS to SCSS